templates/base/footer.tmpl

```diff
@@ -14,8 +14,9 @@
 
 	<script src="{{StaticUrlPrefix}}/js/jquery.js?v={{MD5 AppVer}}"></script>
 	<script src="{{StaticUrlPrefix}}/vendor/plugins/jquery.areyousure/jquery.are-you-sure.js"></script>
-{{if .RequireSimpleMDE}}
-	<script src="{{StaticUrlPrefix}}/vendor/plugins/simplemde/simplemde.min.js"></script>
+{{if .RequireEasyMDE}}
+	<script src="{{StaticUrlPrefix}}/vendor/plugins/easymde/easymde.min.js"></script>
+	<script src="{{StaticUrlPrefix}}/vendor/plugins/codemirror/codemirror.js"></script>
```

Comment:
Interesting. Why did it work without CodeMirror being loaded before?

Reply:
To be perfectly honest I have no clue, but there is an open PR (that looks unlikely to be merged) regarding it.  
https://github.com/Ionaru/easy-markdown-editor/pull/76

Reply:
I see. I think their method of loading CodeMirror would work in webpack but with this method, we need to manually load it.

Reply:
I guess you should swap the order of these script tags thought.